daemon/graphdriver/aufs/aufs.go

```diff
@@ -308,11 +308,17 @@ func (a *Driver) Put(id string) error {
 
 // Diff produces an archive of the changes between the specified
 // layer and its parent layer which may be "".
-func (a *Driver) Diff(id, parent string) (archive.Archive, error) {
+func (a *Driver) Diff(id, parent string, excludes []string) (archive.Archive, error) {
+	// archive.TarWithOptions uses relative path to compare excludes.
+	// So we have to strip the leading '/'.
+	newExcludes := make([]string, len(excludes))
+	for i, ex := range excludes {
+		newExcludes[i] = strings.TrimLeft(ex, "/")
```

Comment:
Perhaps we can check if the paths are absolute in the `postCommit` method of the server API?